### DISCUSSION TITLE
功能建议：上传图片时支持指定不同的相册

### DISCUSSION THREAD

**Discussion by NyaDoo** [lsky-org/lsky-pro#380] [想法]
建议上传图片时可以指定不同的相册进行上传，便于图片分类。
同时可以对不同相册设置不同的访问权限，例如相册A为公开，相册B为私有。
同时在“我的图片”中增加全选功能也有助于增加图片分类的效率。

**Comment by 0xxb** [comment]
第一点可以在设置中指定相册的，如果想在上传页面指定的话，切换的方式不好定义。

**Reply by NyaDoo** [reply]
设置里面的指定是相对永久性的，还是不太方便。
![image](https://user-images.githubusercontent.com/65238336/162126605-c803a7a9-d8d1-4250-bc5c-b4d0913a6e96.png)
在此处右侧增加下拉菜单指定相册（未选择的情况下则按照设置中的默认相册执行）不知是否可行。
或者换个思路，在用户已登录的情况下，将上传功能与我的相册功能合并，直接在我的相册中增加上传功能，用户当前打开相册为哪个就上传到哪个相册，默认打开用户设置中的默认相册。

**Reply by 0xxb** [reply]
后者是可行的。

**Comment by YiHui-Liu** [comment]
全选功能强烈渴望；另外，能否在侧边栏添加相册的快捷入口？

**Reply by 0xxb** [reply]
@YiHui-Liu mac 客户端是可以通过 command+a 全选的，其他的还没有适配，后面会加上。
